.gitmodules

```
@@ -31,3 +31,6 @@
 [submodule "dwave-samplers"]
 	path = dwave-samplers
 	url = https://github.com/dwavesystems/dwave-samplers.git
+[submodule "dwave-gate"]
+	path = dwave-gate
+	url = git@github.com:dwavesystems/dwave-gate.git
```

Should use https, so that it can be checked out anonymously:
```suggestion
	url = https://github.com/dwavesystems/dwave-gate.git
```